pkg/daemon/ceph/client/filesystem_mirror.go

```diff
@@ -79,7 +79,10 @@ func DisableFilesystemSnapshotMirror(context *clusterd.Context, clusterInfo *Clu
 	// Run command
 	output, err := cmd.Run()
 	if err != nil {
-		return errors.Wrapf(err, "failed to disable ceph filesystem snapshot mirror for filesystem %q. %s", filesystem, output)
+		// We cannot wrap here since errors.Wrapf will return *errors.withStack which is not exported so we can't assert it
+		// The call reads *exec.ExitError so won't be work
+		logger.Errorf("failed to disable ceph filesystem snapshot mirror for filesystem %q. %s. %v", filesystem, output, err)
```

Comment:
Not sure if it's worth logging actually, it might be confusing but it's still good to print the output. I'll log in the caller that we ignore the error.